src/components/selection-figure.js

```diff
@@ -15,7 +15,7 @@ let SelectionFigure = createClass({
   },
 
   getTitle(title = '') {
-    let trimmed = title.trim()
+    let trimmed = title ? title.trim() : ''
```

Comment:
I generally try to avoid any logic in the template, even as small as that logic is.

good call on removing the default in the signature. Looks like the new code will handle both `null` and `undefined`